build.gradle.kts

```diff
@@ -46,7 +46,11 @@ dependencies {
 
     // Use the Kotlin JUnit5 integration.
     testImplementation("org.junit.jupiter:junit-jupiter:5.6.0")
+    testImplementation("org.junit.jupiter:junit-jupiter-params:5.6.0")
```

Comment:
Needed to be able to run [parametised tests](https://junit.org/junit5/docs/current/user-guide/#writing-tests-parameterized-tests).